addon/templates/components/popover-on-element.hbs

```diff
@@ -1,3 +1,12 @@
-{{yield (hash
-	hide=(action "hide")
-)}}
+{{#if _shouldRender}}
```

Comment:
Correct me if I'm wrong:

- `_shouldRender` will be `false` by default if `enableLazyRendering` is `true`
- `_shouldRender` will be `true` by default if `enableLazyRendering` is `false`